### DISCUSSION TITLE
What?

### DISCUSSION THREAD

**Discussion by BryanDollery** [hyprwm/Hyprland#8133] [General]
I have a strange "_bug_" (aren't they all?).

I have a weird ass problem. I boot straight into arch/hyprland (no login, just straight in). When I try to run, say, Kitty, by pressing win+t, Hyprland completely crashes and dumps me back to tty. This happens with everything I open via the keyboard.

But, if I press win+space to show Rofi and select Kitty, it all goes as planned. From then on, I can use the keyboard shortcuts without any problems, and it is very stable.

I'm pretty sure this isn't a bug - I've probably just made a mistake somewhere, but I can't find it so if anyone has a clue, please let me know. TIA

**Comment by Aqa-Ib** [comment]
Try installing hyprland-git to see if it is solved there.

If not, file a bug report please with the information required to be able to help you.

**Reply by BryanDollery** [reply]
Hmm, yeah, but... I really don't think this is a bug. I think it's some weird shit I've wired together wrong :) I'm already running the -git version of everything I can get my hands on, including hyprland, waybar, etc.

**Reply by Aqa-Ib** [reply]
If hyprland crashes it is a bug. When did this start to happen? Try installing version 0.44.1 to see if it happens there?

**Reply by BryanDollery** [reply]
Oh. It stopped. I think it was a bug after all, because I've just pulled the latest from hyprland's main branch, and it has stopped. Thanks for taking the time to help.
